src/components/TeamViewItem.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export default class TeamViewItem extends React.Component{
+    constructor(props){
+        super(props);
+    }
```

Comment:
Since you aren't doing anything but the super constructor, this constructor is unnecessary and can be removed. Also, once it's removed, since the only thing happening is the render, it could probably be converted into a stateless function (though that's not necessary yet).